howto/configure-redis/README.md

```diff
@@ -124,7 +124,19 @@ kubectl apply -f redis-state.yaml
 kubectl apply -f redis-pubsub.yaml
 ```
 
-### Standalone
+### Self Hosted Mode
 
 By default the Dapr CLI creates a local Redis instance when you run `dapr init`. However, if you want to configure a different Redis instance, create a `components` dir containing the YAML file and provide the path to the `dapr run` command with the flag `--components-path`.
 
+### Self Hosted Mode without Docker
+
+In this mode, the Dapr CLI does not create a Redis instance. 
+
+#### Option 1: Local Redis Setup
```

Comment:
What are these options referring to? Are these referring to Option 1 and Option 2 earlier in this topic?